vm/sdk/wallet/tx.go

```diff
@@ -28,6 +30,38 @@ func encode(fields ...scale.Encodable) []byte {
 	return buf.Bytes()
 }
 
+func Deploy(pk signing.PrivateKey, nonce core.Nonce, blob []byte, opts ...sdk.Opt) ([]byte, error) {
+	options := sdk.Defaults()
+	for _, opt := range opts {
+		opt(options)
+	}
+	meta := core.Metadata{
+		Nonce:    nonce,
+		GasPrice: options.GasPrice,
+	}
+
+	principal := wallet.ComputePrincipal(signing.Public(pk))
+	var blobEncoded bytes.Buffer
+	if _, err := codec.EncodeByteSlice(&blobEncoded, blob); err != nil {
+		return nil, fmt.Errorf("encoding code blob: %w", err)
+	}
+
+	athPayload := athcon.Payload{
+		Selector: &wallet.DeploySelector,
+		Input:    blobEncoded.Bytes(),
+	}
+	payload, err := gossamerScale.Marshal(athPayload)
+	if err != nil {
+		return nil, fmt.Errorf("encoding tx payload: %w", err)
+	}
+	cPayload := core.Payload(payload)
+	tx := encode(&sdk.TxVersion, &principal, &meta, &cPayload)
+
+	// sig := ed25519.Sign(ed25519.PrivateKey(pk), core.SigningBody(options.GenesisID[:], tx))
```

Comment:
Commented code left. Add a comment explaining why it's there or remove the commented line.

Reply:
It's left in each one (`Spawn`, `Spend` and `Deploy`) - in the future we must add the genesisID prefix for signing the TX.